test/inductor/test_aot_inductor.py

```diff
@@ -16,17 +16,25 @@
 from torch._inductor.utils import aot_inductor_launcher, cache_dir
 
 from torch.testing import FileCheck
+from torch.testing._internal import common_utils
 
 from torch.testing._internal.common_utils import (
     IS_CI,
     IS_FBCODE,
     IS_WINDOWS,
+    skipIfRocm,
     TEST_WITH_ROCM,
     TestCase,
 )
-from torch.testing._internal.inductor_utils import HAS_CUDA
+
+# Defines all the kernels for tests
```

Comment:
Is this comment relevant?

Reply:
That comment is supposed to match `from torch.testing._internal.triton_utils import add_kernel`

Reply:
Ah, ok, should we move it then to avoid confusions like mine? :)